project/Build.scala

```diff
@@ -948,6 +948,7 @@ object Build {
         (
           (dir / "shared/src/test/scala/org/scalajs/testsuite/compiler" ** "IntTest.scala").get
           ++ (dir / "shared/src/test/scala/org/scalajs/testsuite/javalib/lang" ** "IntegerTest.scala").get
+          ++ (dir / "shared/src/test/scala/org/scalajs/testsuite/javalib/lang" ** "ObjectTest.scala").get
```

Comment:
Is there no way to do `IntegerTest.scala OR ObjectTest.scala` instead of duplicating the full path ?

Reply:
Not that I know of. Hopefully it will soon turn into `*.scala`.

Reply:
I think you may be able to do `OrFilter("IntegerTest.scala", "ObjectTest.scala"))`

Reply:
Indeed. Done.